.github/workflows/ci-docker.yml

```diff
@@ -0,0 +1,85 @@
+name: Docker CI
+
+on:
+  pull_request:
+    branches: ['main']
+    paths: ['Dockerfile','.github/workflows/ci-docker.yml']
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: blinklabs/amaru
```

Comment:
This should be `blinklabs-io/amaru` for GHCR, but it doesn't actually cause a problem here since we don't push

Reply:
I copied this from the Mithril client repo. 😅